lib/asset/uploadModel.js

```diff
@@ -5,6 +5,32 @@ const http = require('../util/http.js').func
 exports.required = ['data']
 exports.optional = ['itemOptions', 'asset', 'jar']
 
+// Docs
+/**
+ * Upload a model.
+ * @category Assets
+ * @alias uploadModel
+ * @param {Buffer} data - The model data.
```

Comment:
Buffer does not match the `uploadItem` method nor the [Typescript typings](https://github.com/suufi/noblox.js/blob/master/typings/index.d.ts#L984) - is it a buffer or a Stream?